tests/integration-tests/tests/tags/test_tag_propagation.py

```diff
@@ -111,7 +132,7 @@ def _check_tag_propagation(cluster, scheduler, os, volume_name):
             "expected_tags": (
                 cluster_name_tags,
                 {"parallelcluster:node-type": "Compute"},
-                config_file_tags if scheduler == "slurm" else {},
+                {**config_file_tags, **queue_tags, **compute_resource_tags},
```

Comment:
Why do we have `config_file_tags if scheduler == "slurm" else {}` before but now can remove the check on `if scheduler == "slurm"`?

Reply:
I think we also consider scheduler == "plugin" before